addons/mail/controllers/discuss.py

```diff
@@ -416,7 +416,8 @@ def read_followers(self, res_model, res_id):
                 'is_active': follower.is_active,
                 # When editing the followers, the "pencil" icon that leads to the edition of subtypes
                 # should be always be displayed and not only when "debug" mode is activated.
-                'is_editable': True
+                'is_editable': True,
+                'partner': follower.partner_id.mail_partner_format()[follower.partner_id],
```

Comment:
Just to make sure I fully get it:

Passing awareness of partner of follower is a necessity on chatter because if you make a search and it matches same or equal limit of already known partners, it doesn't RPC and so still not provide what are considered more relevant partners?

Or does the RPC but the immediate awareness is to prompt relevant user immediately without waiting the RPC?

I think it's either case, but which one?... Or if I'm completely wrong can you clarify?

Reply:
I would say both:

1. quicker accurate result: before the return of the RPC, followers will be listed immediately (most of the time we ping someone already follower, so the RPC becomes useless - but it's still done just in case)
2. JS is sorting internal user followers above non-internal users and non-followers, so there is always a chance the RPC would return too many results (eg. typing a single letter), and therefore would not return the follower in question. If we don't know the follow is an internal user, it will not be ranked first after the response of the RPC